styles.css

```diff
@@ -0,0 +1,114 @@
+body {
+  background-color: #FFD9D9;
+  margin: 60px;
+  max-width: 960;
+}
+
+* {
+  font-family: "century gothic"; 
+}
+
+.header--headline {
+  font-size: 32px;
+  font-weight: bold;
+}
+
+.hr--style {
+  border: 0;
+  height: 1px;
+  background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0));
+}
+
+.main-body--hr__top {
```

Comment:
Great first attempt at using BEM.  👍   We should run through this in office hours or after class to just solidify the BEM concepts to make sure it is being applied appropriately.